test/uk/gov/hmrc/plasticpackagingtax/registration/views/OrganisationDetailsTypeViewSpec.scala

```diff
@@ -28,7 +28,7 @@ import uk.gov.hmrc.plasticpackagingtax.registration.forms.OrgType.{
   SOLE_TRADER,
   UK_COMPANY
 }
-import uk.gov.hmrc.plasticpackagingtax.registration.forms.OrganisationType
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.{OrgType, OrganisationType}
```

Comment:
Is import `OrganisationType` still required? We've got stuff such as `.fill(OrganisationType(UK_COMPANY.toString))` in the file but should we be using `OrgType` instead now?

Reply:
Thanks for clarifying :muscle: